docs/reference/ingest/search-inference-processing.asciidoc

```diff
@@ -0,0 +1,185 @@
+[[ingest-pipeline-search-inference]]
+=== Inference processing
+
+Starting in 8.5, *Search* creates ingestion pipelines by default to work with indices created for search use cases.
```

Comment:
Try to re-frame the feature as an Elasticsearch feature by focusing on the specific UI rather than a concept of a "Search" area or solution. For example:

```suggestion
When you create an index through the **Content** UI, a set of default ingest pipelines are also created, including a ML inference pipeline.
```